templates/shelf_jwt_auth/lib/router.dart

```diff
@@ -0,0 +1,127 @@
+// Configure routes.
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:bcrypt/bcrypt.dart';
+import 'package:dart_jsonwebtoken/dart_jsonwebtoken.dart';
+import 'package:shelf/shelf.dart';
+import 'package:shelf_router/shelf_router.dart';
+
+import 'app.dart';
+
+final router = Router()
+  ..post('/register', _register)
+  ..post('/login', _login)
+  ..get('/me', _getCurrentUser);
+
+Middleware authMiddleware = (innerHandler) {
+  return (request) {
+    final reqPath = request.url.path;
+
+    // Bypass auth header check for these routes
+    if (const ['login', 'register'].contains(reqPath)) {
```

Comment:
is there any other way to check if path matches some route? maybe something is exported from shelf itself?

Reply:
Nope, I haven't found any.